trans.py

```diff
@@ -2,6 +2,25 @@
 import struct
 import logging
 
+def read_file():
```

Comment:
```suggestion
def read_configuration_file():
```

Is more clear to indicate what the function is doing